Makefile

```diff
@@ -20,29 +19,23 @@ ifdef CACHE_FROM
 _CACHE_FROM := --cache-from $(CACHE_FROM)
 endif
 
-# Setting SHELL to bash allows bash commands to be executed by recipes.
```

Comment:
same here, this is all part of kubebuilder scaffolding, let's keep these unless there is an issue